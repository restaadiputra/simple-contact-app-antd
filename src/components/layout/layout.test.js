import React from 'react';
import { render } from '@testing-library/react';
import Layout from './layout';

test('should render children', () => {
  const children = <div data-testid="test" />;
  const { container, getByTestId } = render(<Layout>{children}</Layout>);
  expect(getByTestId('test')).toBeInTheDocument()
  expect(container.firstChild).toMatchSnapshot();
});
