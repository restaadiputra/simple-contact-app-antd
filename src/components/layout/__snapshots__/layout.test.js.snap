// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render children 1`] = `
<main
  class="ant-layout-content sc-bdnylx ehLFKO"
>
  <div
    data-testid="test"
  />
</main>
`;
